python-frontend/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -2597,6 +2597,56 @@ public void except_group() {
       .hasMessage("Parse error at line 2: except* clause must specify the type of the expected exception.");
   }
 
+  /**
+   * except* body cannot contain continue, break or return instruction
+   */
+  @Test
+  public void except_group_invalid_instruction() {
+    final String code1 = "try:pass\n" +
```

Comment:
Indeed this is not needed, as without is the variable is still effectively final. I will remove this keyword.

I tried at first to have an unique variable code that I would reuse anywhere like I did for tree/trystatement, but this is of course not possible since it is used in a lambda expression, so I got the error message "Variable used in lambda expression should be final or effectively final".
I didn't think much of it and put it as final and making several version of it (code1, code2, ...)